@xen-orchestra/web-core/lib/components/network/VtsNetworkStatus.vue

```diff
@@ -0,0 +1,54 @@
+<template>
+  <UiInfo class="status-indicator text-ellipsis" :accent="computedStatusProps.accent">
+    <p v-tooltip class="text-ellipsis" :class="{ card }">{{ computedStatusProps.text }}</p>
+  </UiInfo>
+</template>
+
+<script setup lang="ts">
+import UiInfo from '@core/components/ui/info/UiInfo.vue'
+import { vTooltip } from '@core/directives/tooltip.directive'
+import type { IconDefinition } from '@fortawesome/fontawesome-common-types'
+import { faCheck, faExclamation } from '@fortawesome/free-solid-svg-icons'
+import { computed } from 'vue'
+import { useI18n } from 'vue-i18n'
+
+const props = defineProps<{
+  items?: any | any[]
+  card?: boolean
+}>()
```

Comment:
You can use destructured props since vue 3.5  
```suggestion
const { items, card } = defineProps<{
  items?: any | any[]
  card?: boolean
}>()
```